src/main/java/com/algosenpai/app/request/README.md

```diff
@@ -0,0 +1,3 @@
+### Request
```

Comment:
Also what is this for?

Reply:
Like isnt it part of UI?

Reply:
I will remove request and response package

Reply:
removing it